metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableBackend.java

```diff
@@ -390,6 +391,14 @@ public Statistics getStatistics() {
 
         for (final Pair<RowKey, Mutations> e : saved) {
             final ByteString rowKeyBytes = rowKeySerializer.serializeFull(e.getKey());
+
+            if (rowKeyBytes.size() >= MAX_KEY_ROW_SIZE) {
+                reporter.reportWriteDroppedBySize();
+                log.error("Row key length greater than 4096 bytes (2): " + rowKeyBytes.size()
```

Comment:
what's the significance of (2) vs (1) in the logs?

Reply:
This just to indicate where they are coming from in the codebase

Reply:
because one is batch and one is writeOne?

Reply:
yep, I want to make sure in the future we know where they are coming from (metrics and logs I guess)